__tests__/api.test.js

```diff
@@ -164,3 +164,52 @@ describe("GET/api/articles/:articles_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("GET status: 200, responds with an array of comment objects, each of which should have the following properties:comment_id, votes, created_at, author, body, article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBe(11);
+        response.body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
+      });
+  });
+
+  it("GET status: 200, sorts by `created_at` in descending order", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments).toBeSorted({
+          descending: true,
+          key: "created_at",
+        });
+      });
+  });
+
```

Comment:
Missing test - 200 valid article id that has no comments, response.body.comments should be an empty array []

Reply:
You may need a checkExists function to differentiate between valid id with no comments (200 db returns empty rows) and non-existent article with no comments (404 db _also_ returns empty rows)

Drop me a message if you need clarification for this!